docs/introduction/introduction.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: default
+title: Introduction
+nav_order: 3
+has_children: true
+permalink: /introduction
+---
+
+# Introduction
+{: .no_toc }
+
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+---
+
+If you are looking for a gentle introduction to containers and the Open Container
+Initiative, you've come to the right place!
+
+## Containers
+
+A container is generally an encapsulated environment that is created from one or more binaries (images) to provide some form of isolation. Read more about [containers here]({{ site.baseurl }}docs/introduction/container-images/).
+
+## What is OCI?
```

Comment:
I would say omit this section as you have already laid out a basic navigation in the index (and left nav) to be:
- About
- Introductory Material
- Contributing

Reply:
These sections are the topics for the introductory material:

 - Container Images
 - What is OCI
 - Specification Types
 - Content Addressability
 - Distribution

The children of "Introductory material" one level up at:

 - About
 - *Introductory Material*
 - Contributing

I would suggest that we keep an organizational structure under Introductory Material, because it's likely to grow and even need further sub topics. Is there a better title / order for this list, or just a rephrasing of the container image section, so you don't feel it is redundant?

Reply:
I agree that the structure should be essentially

- About
- Introductory Material
  - Container Images
  - What is OCI   --> Links to /About
  - Specification Types
  - Content Addressability
  - Distribution
- Contributing

My comment was more that this one little paragraph "What is OCI" then just linked back to a top level 'About'. Seemingly redundant or circular..    Perhaps there is no top level /About or else we need to figure out what content should go into "What is OCI" that is different than /About

Reply:
Oh, good call! You are saying that "About this web interface thing you are looking at" is a different beastie than "About OCI under the docs." I completely agree. Right now it links to the "About this site" page, and I think we should likely have a more verbosely written "This is what OCI is all about, history" under the docs. I think this detail is out of scope for this first go, but important!